### PR TITLE
Development

### PR DESCRIPTION
**Pull request by EXECNOIR**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,8 +55,58 @@ Project is created with:
...
+ - In CLion, go to Settings -> Build, Execution, Deployment -> Toolchains
+ - At the environment field, locate the mingw-w64 you built OpenCV and SQLite3 with
+ - Fork and clone the repository
+ - Open the project folder in CLion with File -> Import Project and then click on Open existing CMake project
+ - CLion should generate a folder in the projet root, called cmake-build-debug
+ - In your OpenCV install directory, locate ./install/x64/mingw/bin
+ - Copy all the DLL files to the cmake-build-debug folder and the gtest folder found in cmake-build-debug
+ - Enjoy...
 
 ## HOW TO USE?
```
Comment (leventekobor) [inline README.md]: this will stay embty for a time?

Comment (leventekobor) [reply README.md]: also no need for empty points ( License, Having trouble...)

File: src/initial.cpp [hunk 2]
```diff
@@ -8,4 +8,4 @@
 int initialFunction(int a, int b)
```
Comment (leventekobor) [inline src/initial.cpp:8]: I think it is time to remove this

File: src/database_handler.h [hunk 3]
```diff
@@ -6,8 +6,7 @@
 #include <cstring>
 #include <opencv2/core/mat.hpp>
 #include <opencv2/opencv.hpp>
-//#include "../../../../../../sqlite3/include/sqlite3/sqlite3.h"
-
+//#include "../../../../../../sqlite3/include/sqlite3/sqlite3.h" // I need this, please don't delete it - F
```
Comment (leventekobor) [inline src/database_handler.h]: if you need this why it is commented out?

File: docs/functions.md [hunk 4]
```diff
@@ -84,10 +84,10 @@
```
Comment (leventekobor) [inline docs/functions.md:84]: what are all the `|` in the md file?

**Review by leventekobor — CHANGES_REQUESTED** [review]
This is actually a pretty small PR

File: src/database_handler.h (hunk 3 above)
Comment (VeghFruzsi) [reply src/database_handler.h]: SQL works me just with that link, but for the boys work with the short one.  I have just windows 8.1 from the group so we think this is the problem. But with that works perfectly.

Comment (leventekobor) [reply src/database_handler.h]: okay i see :)

**Review by leventekobor — COMMENTED** [review]
Why only 7 files changed? What happened?

**Pull request closed without merging**